### ISSUE TITLE
MatSelect: dropdown arrow renders as rectangle in high-contrast mode

### ISSUE THREAD

**Issue by ankz2390** [angular/components#14207]
Bug, feature request, or proposal:
Bug

What is the expected behavior?
Dropdown arrow should be visible on windows HC mode

What is the current behavior?
Dropdown icon not visible. Replaced with a small box

What are the steps to reproduce?
Activate windows HC mode using accessibility menu, 

What is the use-case or motivation for changing an existing behavior?
Catering for accessibility for disabilities requiring high contrast modes

Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Edge

![image](https://user-images.githubusercontent.com/3615305/48742301-3e62db80-ec13-11e8-8ce2-cf6ccd7d41b6.png)

**Comment by amysorto** [comment]
This has been resolved in the new MDC based select component coming in v15 soon
